### PR TITLE
Branch 2

### PR DESCRIPTION
**Pull request by justynapaldyna**

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  # GET /users/1/edit
+  def edit
+  end
+
+  # POST /users or /users.json
+  def create
+    @user = User.new(user_params)
+
+    respond_to do |format|
```
Comment (paleksandrowicz) [inline app/controllers/users_controller.rb:27]: Why do we respond to html and json?

Comment (justynapaldyna) [reply app/controllers/users_controller.rb:27]: I used the scaffolding method, i wrote it on Trello! Not sure if i can leave the controller like that, should I write it by myself? By now, I think i can just delete format.json - we don't need potential javascript for client, right?

Comment (justynapaldyna) [reply app/controllers/users_controller.rb:27]: As i understand, render_to do format.html is a method similar to flash notifications that were introduced in a course, so i would leave format.html and delete format.json as they are not nessesary :D

Comment (paleksandrowicz) [reply app/controllers/users_controller.rb:27]: Yeah I would remove the json format for now.

Comment (justynapaldyna) [reply app/controllers/users_controller.rb:27]: Thanks Peter

**Pull request merged**